### DISCUSSION TITLE
【喜报】绳网帖子能无限刷新了！

### DISCUSSION THREAD

**Discussion by user1-cloud** [share121/inter-knot#1041] [其他]
[![image](https://github.com/user-attachments/assets/6f299991-c5d1-4cc9-bb22-3b1594aeecd1)](https://www.bilibili.com/video/BV1GJ411x7h7/?share_source=copy_web&vd_source=722b2d6c4a0efb2b0fc1aa3c34f95a9d)
技术爆炸了（）

**Comment by BITE-7274** [comment]
怎么做到的·-·求

**Comment by shilijia-siweikongjian** [comment]
同求

**Comment by user1-cloud** [comment]
> 怎么做到的·-·求
***
> 同求
***
@BITE-7274 @shilijia-siweikongjian 额，我的意思是之前绳网上最多只会显示100条帖子，现在它更新了可以无限显示下去了
至于我这个图，是通过edge的缩放功能截出来的
